web/src/features/hfiCalculator/components/DayHeaders.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { Table, TableBody, TableContainer, TableRow } from '@material-ui/core'
+import { render } from '@testing-library/react'
+import React from 'react'
+import DayHeaders from 'features/hfiCalculator/components/DayHeaders'
+describe('DayHeaders', () => {
+  it('should return table row with the headers for Monday - Friday given the ISO Date', () => {
+    const prepCycle = [
+      'Mon., Oct. 04',
```

Comment:
Will this fail when run with different locale settings? If your local machine is french, what will this work?